setup.py

```diff
@@ -12,6 +12,11 @@
 import sys
 import sysconfig
 from wheel.bdist_wheel import bdist_wheel
+
+# Override the default MACOSX_DEPLOYMENT_TARGET for the build. If not set, it will use the
+# default value from python sysconfig.get_config_var('MACOSX_DEPLOYMENT_TARGET').
+MACOS_DEPLOYMENT_TARGET_OVERRIDE = 10.15
+
```

Comment:
I think we need to set the environment variable so that pypi can pick it up. Setting it only for C code won't work.